config/src/config/network_config.rs

```diff
@@ -140,10 +140,10 @@ impl NetworkConfig {
                 self.network_peers.peers.is_empty(),
                 "Validators should not define network_peers"
             );
-        } else if self.is_permissioned {
+        } else if self.enable_remote_authentication {
```

Comment:
there's actually no way to enforce ids outside of the other peer enforcing remote authentication, we should consider deleting this piece of code...  we should consider deleting this code. cc: @phlip9

Reply:
As discussed offline, we'll address these logic changes in a future PR. This PR is just for renaming.